pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1663,3 +1663,331 @@ def func(x):
     msg = "length must not be 0"
     with pytest.raises(ValueError, match=msg):
         df.groupby("A", observed=False).agg(func)
+
+
+def test_agg_simple_lambda_numpy_to_same_data_type():
+    df = DataFrame(
```

Comment:
Add GH# for each test - if there isn't a relevant issue, then add this PR's number instead.